<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TickEditingWidget</class>
 <widget class="QWidget" name="TickEditingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="PositionChoiceWidget" name="positionChoiceWidget" native="true"/>
     </item>
     <item>
      <widget class="QCheckBox" name="internalTicksEnabledCheckBox">
       <property name="text">
        <string>internal ticks</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="PenChoiceWidget" name="penChoiceWidget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="subTickCountLabel">
       <property name="text">
        <string>Sub tick count</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="subTickCountSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Length in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lengthInDoubleSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Length out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lengthOutDoubleSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PenChoiceWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/penchoicewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PositionChoiceWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/positionchoicewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
